src/backend/shortcuts/nonesteamgame/nonesteamgame.ts

```diff
@@ -18,11 +18,17 @@ import {
   generateShortAppId,
   removeImagesFromSteam
 } from './steamhelper'
-import { app } from 'electron'
+import { app, BrowserWindow } from 'electron'
 import { isFlatpak, isWindows, tsStore } from '../../constants'
 import { logError, logInfo, LogPrefix, logWarning } from '../../logger/logger'
 import i18next from 'i18next'
 import { showErrorBoxModalAuto } from '../../dialog/dialog'
+import { GlobalConfig } from '../../../backend/config'
```

Comment:
This relative import doesn't look quite right

Reply:
hmmm this is the only way I can get this to work

Reply:
I'm leaving this like this for now since I can't find any other way to reference this file without an error, it will be easier to fix once the PR changing the import config is merge (then it will just be `backend/config`)